sycl/source/detail/scheduler/scheduler.hpp

```diff
@@ -444,9 +450,13 @@ class Scheduler {
   const QueueImplPtr &getDefaultHostQueue() const { return DefaultHostQueue; }
 
   static MemObjRecord *getMemObjRecord(const Requirement *const Req);
+  // Virtual for testing purposes only
+  void deferMemObjRelease(const std::shared_ptr<detail::SYCLMemObjI> &MemObj);
 
   Scheduler();
   ~Scheduler();
+  void releaseResources();
+  inline bool isDeferredMemObjectsEmpty();
```

Comment:
Could you please clarify why `inline` is needed here?

Reply:
fixed in https://github.com/intel/llvm/pull/6837/commits/71e9048af88ddcc853968cf9bbaa050f63ad323e
you are right, compiler will decide